typescript-sdk/src/observability-sdk/instrumentation/langchain/__tests__/integration/simple-agent-and-tool.integration.test.ts

```
@@ -0,0 +1,635 @@
+      expect(llmSpan?.name).toMatch(/openai gpt-4o-mini \(temp 0\.7\)/);
+      expect(llmSpan?.attributes["gen_ai.request.model"]).toBe("gpt-4o-mini");
+      expect(llmSpan?.attributes["gen_ai.request.temperature"]).toBe(0.7);
+    });
+
+    it("should name tool spans with tool name and input preview", async () => {
+      const tools = [
+        new DynamicTool({
+          name: "calculator",
+          description: "Perform mathematical calculations",
+          func: async (input: string) => {
+            return `Result: ${eval(input)}`;
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)